chanzuckerberg/single-cell-data-portal
# fix: Staging hotfix 08/18/23

## Reason for Change

- #5555 

## Changes
- Updating batch job memory limits for upload jobs
- Fixing input datatype error when passing into downstream publish function

## Testing steps
- Completed an end-to-end rdev migration with a subset of datasets
- After merging, I will test a partial migration in staging (i.e. will stop once I confirm publish step works for at least one collection)

## Notes for Reviewer
